### PR TITLE
Bkammin/foundations cleanup

### PR DESCRIPTION
**Pull request by bkammin**
This PR encompasses the following JIRA tickets related to Foundations Cleanup:

https://jira.wnyc.org/browse/DS-634
https://jira.wnyc.org/browse/DS-644
https://jira.wnyc.org/browse/DS-646
https://jira.wnyc.org/browse/DS-650
https://jira.wnyc.org/browse/DS-640
https://jira.wnyc.org/browse/DS-649
https://jira.wnyc.org/browse/DS-645
https://jira.wnyc.org/browse/DS-642
https://jira.wnyc.org/browse/DS-647

### CODE REVIEW

File: app/styles/_library/_base.forms.scss [hunk 1]
```diff
@@ -54,10 +54,12 @@ input[type=checkbox] {
   cursor: pointer;
   display: block;
   float: left;
-  border: 2px solid rgb(var(--color-border-bold));
+  border: 2px solid rgb(var(--color-gray), 0.3);
```
Comment (bkammin) [inline app/styles/_library/_base.forms.scss:57]: colors question: i'm replacing a color token with a specific color here. is there a color token i should use instead, or do we need to update `--color-border-bold`?

File: app/styles/_library/_base.forms.scss [hunk 2]
```diff
@@ -54,10 +54,12 @@ input[type=checkbox] {
...
   padding: 0;
   user-select: none;
   appearance: none;
+  -webkit-appearance: none;
```
Comment (bkammin) [inline app/styles/_library/_base.forms.scss:61]: seems like there should be a mixin to do this

File: app/styles/_library/_base.forms.scss (hunk 1 above)
Comment (kimlarocca) [reply app/styles/_library/_base.forms.scss:57]: we'll make a note of this for when we finish up the colors

File: app/styles/_library/_base.forms.scss (hunk 2 above)
Comment (kimlarocca) [reply app/styles/_library/_base.forms.scss:61]: We should probably do a styles cleanup at one point, go through an optimize things. I'll make a note to do a ticket for this so we don't forget!

**Pull request merged**